real_estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,31 @@
+from odoo import models, fields
+
+class EstatePropertyOffer(models.Model):
+
+    _name = 'estate.property.offer'
+    _description = 'Offers on Property'
+
+    price = fields.Float(
+        string="Offer Price"
+    )
+
+    status = fields.Selection(
+        selection = [
+            ('accepted', 'Accepted'),
+            ('refused', 'Refused')
+        ],
+        string = "Status",
```

Comment:
```suggestion
        string="Status",
```
When passing parameters we don't need to give space in between